testapp/main.go

```diff
@@ -5,21 +5,51 @@ import (
 
 	"github.com/ditrit/verdeter"
 	"github.com/ditrit/verdeter/models"
+	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
-var myCommand = verdeter.NewVerdeterCommand(
-	"math", // the name of the command
-
-	"math app is a test application for verdeter",
-
-	``,
+var myCommand = verdeter.BuildVerdeterCommand(verdeter.VerdeterConfig{
+	Use:         "math",
+	Long:        "math app is a test application for verdeter",
+	Annotations: map[string]string{},
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		println("PersistentPreRun")
+	},
+	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+		println("PersistentPreRunE")
```

Comment:
use log or whatever but this is not understanble...

Reply:
removed